install.sh

```diff
@@ -280,8 +280,10 @@ libsecp256k1_build()
         --enable-benchmark=no \
         MAKE=$make
     $make
-    if ! $make check; then
+    if [[ $use_secp_check == '1' ]] && ! $make check; then
         return 1
+    else
+        echo "Skipping libsecp256k1 tests."
```

Comment:
Did I misunderstand the script syntax, or wouldn't this print "skipping tests" when the tests pass?

Reply:
You're right, fixed.